sources/packages/backend/apps/api/src/auth/decorators/userToken.decorator.ts

```diff
@@ -2,7 +2,7 @@ import { createParamDecorator, ExecutionContext } from "@nestjs/common";
 import { IUserToken } from "../userToken.interface";
 
 export const UserToken = createParamDecorator(
-  (data: unknown, ctx: ExecutionContext) => {
+  (_data: unknown, ctx: ExecutionContext) => {
```

Comment:
if it's not used, do we need to specify it?

Reply:
Yes as we need to access the second parameter here.

Reply:
my qn was about the first parameter, will it work without first parameter

Reply:
No, without the first parameter, ctx will become the first parameter which will not be having execution context in runtime . 

Btw, this is not actually part of PR or story. I just saw the data to be showing warning as it is not used, so tried fixing it.